src/jobs.ts

```diff
@@ -0,0 +1,16 @@
+import { sendSlackSummary } from "./slack";
+import { getCommitDiagnoses } from "./db/models/CommitDiagnosis";
+import { getCommits } from "./db/models/Commit";
+import { SLACK_DAY_INTERVAL } from "./config";
+
+const dateOffset = 86400000 * SLACK_DAY_INTERVAL; // day interval in milliseconds: 24 * 60 * 60 * 1000 * DAY_INTERVAL
```

Comment:
We can use two variables to be clearer:
- const for milliseconds in a day
- const for the duration (in milliseconds) of summarized info